controlplane/eks/api/v1beta1/awsmanagedcontrolplane_types.go

```diff
@@ -167,6 +168,10 @@ type AWSManagedControlPlaneSpec struct { //nolint: maligned
 	// +kubebuilder:default=false
 	DisableVPCCNI bool `json:"disableVPCCNI,omitempty"`
 
+	// VpcCni defines a list of environment variables to apply to the `aws-node` DaemonSet
```

Comment:
This is added right now for use case of aws-node DaemonSet but can be used for other env configuration in VPC CNI, so the comment should be generic

Reply:
Good point @Ankitasw :+1: 

Perhaps something like `// VpcCni is used to set configuration options for the VPC CNI (i.e. aws-node daemonset) such as environment variables.`

@charlie-haley - any chance you could change it to something a bit more generic and generate the crd definition again? After that, i think we are good to go.